### PR TITLE
Add Accept-Language and Content-Language headers

### PR DESCRIPTION
**Pull request by wetneb**
For #52.

There seems to be interest in #52 to have more than that: also introduce parameters in the queries to specify the language of the data. I am not too sure how useful that is and which form it should take (do you want to specify the language of each string in the query independently? should we follow JSON-LD conventions to represent strings with languages?) so I am leaving this for another PR.

### CODE REVIEW

File: latest/index.html [hunk 1]
```diff
@@ -455,6 +455,17 @@ <h3>Cross-Origin Access</h3>
...
       </section>
+      <section>
+        <h3>Language Selection</h3>
+        <p>
+           Following [[RFC9110]], services SHOULD use the <code>Accept-Language</code> header to let clients select the language in which user-facing text is returned.
+           They SHOULD use the <code>Content-Language</code> in their responses to expose the language each response is returned in.
+        </p>
+        <p>
+          Examples of user-facing text in service responses are: the name of the service and the name of property configuration fields in the manifest, the name and description of entities, types and properties,
+          the contents of the entity preview pages.
```
Comment (tfmorris) [inline latest/index.html:466]: This should probably also control what URL is returned for the `documentation` URL in the manifest (unless some other mechanism is proposed for multilingual documentation).

**Review by tfmorris — CHANGES_REQUESTED** [review]

File: latest/index.html (hunk 1 above)
Comment (wetneb) [reply latest/index.html:466]: How about the same mechanism? The documentation URL itself could already react to `Accept-Language`, no? Isn't that how most websites (should) work?

Comment (tfmorris) [reply latest/index.html:466]: That sounds fine.

Comment (fsteeg) [reply latest/index.html:466]: So should we add the documentation here, like this:

```suggestion
          the contents of the entity preview pages, and the documentation linked in the manifest.
```

**Review by fsteeg — APPROVED** [review]

**Comment by wetneb** [comment]
Given https://github.com/reconciliation-api/specs/issues/52#issuecomment-1558986118 I would propose to remove the `Content-Language` header.

**Review by tfmorris — APPROVED** [review]

**Pull request merged**